src/Liuggio/StatsdClient/Entity/StatsdData.php

```diff
@@ -1,102 +1,159 @@
-<?php
```

Comment:
There's line errors, can you remove them ? It is quite disturbing to get 1043 additions for such a simple PR :)